ballerina-tests/tests/60_client_send_error_in_client_bidi_streaming_client.bal

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/grpc;
+import ballerina/test;
+import ballerina/lang.runtime;
+
+@test:Config {enable:true}
+isolated function testClientStreamingSendErrorToService() returns grpc:Error? {
+    ErrorSendServiceClient errorClient = check new ("http://localhost:9160");
+    SendErrorClientStreamingStreamingClient streamingClient = check errorClient->sendErrorClientStreaming();
+
+    check streamingClient->sendString("Hey");
```

Comment:
Here, I don't understand the intention behind these tests. You are sending an error first and then create a new client and send correct messages again. Don't we need to reuse the same streaming client here?

Reply:
Im trying to check whether the service received an error in the previous client connection

Reply:
Okay, got it.